client/my-wallet.js

```diff
@@ -13,7 +13,7 @@ const space = storage.space({ signer })
 
 async function main() {
   const spaceObject = {
-    controller: signer.id,
+    controller: signer.controller,
```

Comment:
This returns a DID like `did:key:{pk}` not a DID URI to the verificationMethod in the did doc which has uri `did:key:{pk}#{pk}`.

I think it's best to just a bare DID.